CONTRIBUTING.md

```diff
@@ -183,15 +183,15 @@ All pull requests must be approved by a *collaborator* before the pull request c
 
 Before a pull request is accepted, time should be given to receive input from *collaborators* or *code owners* with the expertise to evaluate the changes. The amount of time can vary but at least 3 days during the typical working week and 5 days over weekends should be given to account for international time differences and work schedules.
 
-When a pull request : (a) has a significant impact on the project, (b) is inherently controversial, or (c) has not reached consensus with *collaborators*; add a "controversial" label to the pull request for the *steering committee* to review the pull request. Pull requests labeled with "controversial" are not approved until the *steering committee* reviews the issue and makes a decision.
+When a pull request : (a) has a significant impact on the project, (b) is inherently controversial, or (c) has not reached consensus with *collaborators*; add a "status:controversial" label to the pull request for the *steering committee* to review the pull request. Pull requests labeled with "status:controversial" are not approved until the *steering committee* reviews the issue and makes a decision.
```

Comment:
This label may also be great to pair with a GH Discussion...

Reply:
I think that it might be better to keep the discussion directly in the Issue or PR so that it can be managed and discussed in one place. It might be difficult if one group of folks is discussing in the issue and then another group is in Discussions.